src/mumble/Plugins.cpp

```diff
@@ -208,6 +195,16 @@ Plugins::Plugins(QObject *p) : QObject(p) {
 		fPosition[i]=fFront[i]=fTop[i]= 0.0;
 	QMetaObject::connectSlotsByName(this);
 
+	// Initialize strings
+	msContext.len = 256;
+	msContext.data = contextBuf;
+	mwsIdentity.len = 256;
```

Comment:
Are we sure 256 is what we want to limit us to here? Afaik this wasn't a limit before.

Reply:
Indeed. Any suggestions?

Reply:
Dunno. With the C interface structured as it is we will have to set some limit. 256 matches what our link data structure supports to given that bf2.cpp creates a precedence of having complex json state data in the identity we probably should give us some space here. 1024 or 2048 maybe? We don't want it to be to expensive to transmit.

Reply:
I used 256 because it's the limit we set in Link.